code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -60,28 +87,50 @@
 	loc = P
 	card = P
 	sradio = new(src)
-	if(card)
-		if(!card.radio)
-			card.radio = new /obj/item/device/radio(card)
-		radio = card.radio
+	if(!radio)
+		radio = new /obj/item/device/radio/borg(src)

```

Comment:
why are you giving it a borg radio, pai's don't have AI laws and don't get traitorized when the AI does